### PR TITLE
BD-1028:Compliance Issues in Lightning gh-pages repo

### PR DESCRIPTION
**Pull request by mallikarjuna54**

### CODE REVIEW

File: css/prism.css [hunk 1]
```diff
@@ -1,14 +1,3 @@
-/**
```
Comment (mhughesacn) [inline css/prism.css:1]: Please leave this header alone.

File: js/codetabs.js [hunk 2]
```diff
@@ -1,11 +1,3 @@
-/**
```
Comment (mhughesacn) [inline js/codetabs.js:1]: Please leave alone.

File: js/codetabs.js [hunk 3]
```diff
@@ -1,11 +1,3 @@
...
- * Copyright (c) 2017-present, Facebook, Inc.
```
Comment (mhughesacn) [inline js/codetabs.js:2]: Please leave alone.

File: js/scrollSpy.js [hunk 4]
```diff
@@ -1,10 +1,3 @@
-/**
```
Comment (mhughesacn) [inline js/scrollSpy.js:1]: Please revert

**Review by mhughesacn — CHANGES_REQUESTED** [review]
Hi Mallikarjuna - Please leave the existing facebook copyrights alone.  I am looking for Metrological headers on files under js/lightning/src and no more.  Thanks.

**Review by mhughesacn — APPROVED** [review]

**Comment by mhughesacn** [comment]
Thanks Mallikarjuna.  Metrological will have to approve.

**Comment by pradeeptakdas** [comment]
Hi @wouterlucas 
Could you please merge this PR?

**Pull request merged**